common/country_formation/00_major_formables.txt

```diff
@@ -49,7 +49,8 @@ ITA = {
 					country_has_primary_culture = cu:north_italian
 					country_has_primary_culture = cu:south_italian				
 			}
-			has_technology_researched = pan-nationalism	
+			has_technology_researched = pan-nationalism
+			percent = 0.5
 		}
```

Comment:
Perhaps also make a check on the unifier having the tech? I know, they are very unlikely to *not* have it at this point, but still..